examples/dependencies/tests/acceptance/smoke-test.js

```diff
@@ -0,0 +1,17 @@
+import { module, test } from "qunit";
+import { visit, currentURL } from "@ember/test-helpers";
+import { setupApplicationTest } from "ember-qunit";
+
+module("Acceptance | smoke test", function (hooks) {
+  setupApplicationTest(hooks);
+
+  test("visiting /", async function (assert) {
+    await visit("/");
+    let faIcons = document.querySelectorAll(".font-awesome-paragraph svg");
+    assert.equal(faIcons.length, 3, "it renders the fa icons");
+
+    assert
+      .dom("[data-highcharts-chart]")
+      .exists("it initialises the highcharts chart");
+  });
+});
```

Comment:
Should there be a test for the chart as well in case that fails for some reason?

Reply:
`data-highcharts-chart` is added by highcharts when it initialises on the element do you think this is sufficient?

Reply:
Yes, seems good to me - I think we really only need a smoke test that breaks if this doesn't work at all for some reason 👍